Plugins/Flow.Launcher.Plugin.Explorer/Views/ExplorerSettings.xaml.cs

```diff
@@ -123,8 +121,8 @@ private void expActionKeywords_Click(object sender, RoutedEventArgs e)
 
         private void expActionKeywords_Collapsed(object sender, RoutedEventArgs e)
         {
-            if (!expActionKeywords.IsExpanded)
```

Comment:
why is this condition removed?

Reply:
Yeah because this method is hooked up to the ui which triggers when collapsed, so this condition is obsolete.